### PR TITLE
 FP-666: Update docs

### PR DESCRIPTION
**Pull request by mcortes19**
# Ticket(s)

- [FP-666: Docs update](https://fourkitchens.clickup.com/t/36718269/FP-666)

## Purpose

**This PR does the following:**

- Updates project documentation

### Pull Request Deployment

- (optional) Document any unique steps required to deploy this PR (if any)
- Example: Once PR is merged and deployed to remote environment go to [page link] and manually sync data from Live25.

### Functional Testing

- [ ] Open up the `README.md` file
- [ ] Confirm that documentation has been updated correctly

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-666 Updating readme for Shared Component](https://app.clickup.com/t/36718269/FP-666)

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ React Shared Components Library System for Forcepoint. Built on Vite.
 
 ## Library reference
 
+This section provides a concise overview of the key classes and functions available in our library, aiding developers in understanding and utilizing the provided functionalities effectively.
+
 ### Include the Tailwind preset
```
Comment (josue2591) [inline README.md:20]: What do you think of including this line after line 58?

File: docs/ATOMIC_DESIGN.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Atomic Design System
+
+This document describes the structure of the `src/lib/components` folder for creating new Components
+
+## Overview
+
+Welcome to our Atomic Design System! This system is built on the principles of Atomic Design, which organizes user interfaces into discrete levels of complexity—Atoms, Molecules, Organisms, and beyond.
```
Comment (josue2591) [inline docs/ATOMIC_DESIGN.md:7]: I have the feeling that we are not exactly using atomic design here? Does that matter?

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+For publising a new Component, you need add the component in the `src/lib/components/index.ts` file.
+
+Once you run `npm run yalc-publish`, a `dist` folder is created with all components exported.
+
+### Coding standards
+
+- Each Visual *Component* should have an Storybook representation associated.
+- Each *Component* should have RTL support using [TailwindCSS](https://tailwindcss.com/).
+- *Components* can be imported in any React Framework and work properly.
+- Some *Componets* might need to be flexible and accept specifics React Framework component that can override functionalities like [`next/link`](https://nextjs.org/docs/pages/api-reference/components/link) or [`next/image`](https://nextjs.org/docs/pages/api-reference/components/image)
```
Comment (josue2591) [inline docs/CONTRIBUTING.md:24]:
```suggestion
- Some *Components* might need to be flexible and accept specifics React Framework component that can override functionalities like [`next/link`](https://nextjs.org/docs/pages/api-reference/components/link) or [`next/image`](https://nextjs.org/docs/pages/api-reference/components/image)
```

**Review by josue2591 — APPROVED** [review]
Couple of questions and one typo suggestion, other than that this looks great!

File: README.md (hunk 1 above)
Comment (mcortes19) [reply README.md:20]: We can leave it as it is for now, we can improve the documentation in the future

File: docs/ATOMIC_DESIGN.md (hunk 2 above)
Comment (mcortes19) [reply docs/ATOMIC_DESIGN.md]: @josue2591 you are right, I just updated it

**Pull request merged**